doc/configuration.rst

```diff
@@ -400,6 +400,27 @@ Some modules may allow more than one threshold to be defined.  If all the thresh
         }
     }
 
+.. note::
+    New in version 3.17
+
+You can specify ``invisible`` color to hide the block. In this example, loadavg
+will invalidate the block when ``1avg`` (i.e., 12.4) is less than 20 percent.
```

Comment:
Is `invalidate` a better color word over `invisible`?

Reply:
to my non-native-speaker ears: no 😄 

I'd understand "invalidate" as "reset to default color"